src/reducers/versions.spec.tsx

```diff
@@ -1618,4 +1719,94 @@ describe(__filename, () => {
       );
     });
   });
+
+  describe('abortFetchVersion', () => {
+    it('sets the version info to `null` on abortFetchVersion()', () => {
+      const versionId = 123;
+      const state = reducer(
+        undefined,
+        actions.abortFetchVersion({ versionId }),
+      );
+
+      expect(state.versionInfo[versionId]).toEqual(null);
+    });
+
+    it('preserves other version info', () => {
+      const version = fakeVersion;
+      let state = reducer(undefined, actions.loadVersionInfo({ version }));
+
+      const abortFetchVersionId = version.id + 123;
+      state = reducer(
+        state,
+        actions.abortFetchVersion({ versionId: abortFetchVersionId }),
+      );
+
+      expect(state.versionInfo[version.id]).toEqual(expect.any(Object));
```

Comment:
Why are you using `any(Object)` here rather than `createInternalVersion(version)`? You are checking that the other version wasn't set to `null` but not necessarily that it was untouched in any way.